emscripten/webarkitJsfeat.cpp

```diff
@@ -149,13 +149,17 @@ void train_orb_pattern_internal(const char* filename) {
         ext);
     free(ext);
   }
+  webarkitLOGi("Image done!");
+
+  JSLOGi("Starting detection routine...");
```

Comment:
tThese two printings works, they print tese messages:
```c++

Image done!
Starting detection routine...
```
 but at the end of the code they fails to print in the console, i would understand why this happens.... see the comment above.